client/src/mods/Character.lua

```diff
@@ -667,12 +666,7 @@ function Character.fillInMissingSounds(self, sfxTable,  name, maxIndex)
     if sfxTable and sfxTable[i] then
       fillUpSound = sfxTable[i]
     else
-      if i >= perSizeSfxStart[name] then
-        sfxTable[i] = fillUpSound
-      else
-        -- leave it empty
-        sfxTable[i] = nil
-      end
+      sfxTable[i] = fillUpSound
```

Comment:
To me this change feels like a smell.
The indices in that area should never get accessed in the first place so setting them only seems suitable to hide bugs instead of guaranteeing the feature to be working.